src/PolymathError.ts

```diff
@@ -9,6 +9,7 @@ export const ErrorMessagesPerCode: {
     'The wallet is locked, if Metamask extension is being used, the user needs to unlock it first',
   [ErrorCodes.UserDeniedAccess]: 'The user denied access',
   [ErrorCodes.TransactionRejectedByUser]: 'The user rejected the transaction',
+  [ErrorCodes.UnexpectedReturnData]: 'Unexpected return data',
```

Comment:
This error message should maybe be more explicit, something like "The data returned by the smart contract has an unexpected format. Please report this issue to the Polymath team."

Hopefully this error would never reach an end user, but in case it does, it's better for them to know that there is nothing they can do to fix it and the solution lies on our end.